parser.go

```diff
@@ -32,7 +40,7 @@ func ParseMetric(packet []byte) (*Metric, error) {
 	}
 
 	// Create a new hasher for accumulating the data we read
-	h := fnv.New32()
+	h := fnv.New32a()
```

Comment:
noll's recommendation http://www.isthe.com/chongo/tech/comp/fnv/#FNV-1a and for consistency with our use of 1a in hyperloglog. Didn't bother bumping up the digest size since it's probably not relevant (taking remainder over worker count), but I can add that too if you prefer being more consistent.